.travis.yml

```diff
@@ -20,6 +20,7 @@ before_install:
 install:
   - pip install pip --upgrade
   - pip install wheel
+  - pip install nltk
```

Comment:
Why?  nltk is already included in requirements.txt

Reply:
Because we call it directly below.  If it's required at this level, it should be installed at this level.